CONTRIBUTING.md

```diff
@@ -25,16 +40,51 @@ Use these guidelines to contribute to the DC/OS documentation:
                ```
                ---
                post_title: The Title
-               nav_title: The Title
-               menu_order: 1
                ---
                Post markdown goes here.
                ```
 
     **Tip:** There are templates available for some content types. For example, if it's a tutorial you can copy [templates/tutorial.md](templates/tutorial.md) into `foo/` and rename it to `foo/index.md`. Adapt the sections in your new `foo/index.md` to the specifics of your content.
 1. Add images in a child `foo/img/` directory.  
 1. Include all required assets in your `/foo` directory, for example, Marathon app spec, JSON docs, or a Dockerfile.
-1. When you're done, submit a [pull request](https://help.github.com/articles/using-pull-requests/) to the original repo, `dcos-docs`.
-1. For all contributions that include hands-on instructions, such as found in `usage/` or `administration/`, the community managers will test-drive and validate before merging. They might come back to you asking you to fix things. All communication strictly via your pull request on GitHub.  
+
 
 If you're unsure about what exactly should go into the tutorial, you can always check out [spark/](/1.7/usage/tutorials/spark/) for reference.
+
+## <a name="test-local"></a>Test your content locally
+
+1. [Create a repo fork in GitHub](https://guides.github.com/activities/forking/)
+1. [Clone the dcos/dcos-website repo](https://help.github.com/articles/cloning-a-repository/)
+1. Add repo fork as remote repo:
+
+    ```
+    git remote add fork https://github.com/<github-user>/dcos-website
+    git fetch fork
+    ```
+1. Checkout the develop branch:
+
+    ```
+    git checkout develop
+    ```
+1. Update the dcos-docs submodule:
+
+    ```
+    git submodule update --init --recursive
+    ```
+1. [Install Node](https://docs.npmjs.com/getting-started/installing-node)
+1. Install dependencies:
+
+    ```
+    npm install
+    ```
+1. Launch local dev server:
+
+    ```
+    npm start
+    ```
+    (opens dev server in browser)
+    
```

Comment:
We need the instructions that explain how to see what's current in your `dcos-docs` checkout.